docker/Dockerfile.mipsel-unknown-linux-uclibc-cross

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY common.sh lib.sh /
+RUN /common.sh
+
+COPY cmake.sh /
+RUN /cmake.sh
+
+COPY xargo.sh /
+RUN /xargo.sh
+
+ARG VERBOSE
+COPY crosstool-ng.sh /
+COPY cross-toolchains/docker/crosstool-config/mipsel-unknown-linux-uclibc.config /
+RUN /crosstool-ng.sh mipsel-unknown-linux-uclibc.config 5
+
+ENV PATH /x-tools/mipsel-unknown-linux-uclibc/bin/:$PATH
+
+COPY deny-debian-packages.sh /
+RUN TARGET_ARCH=mipsel /deny-debian-packages.sh \
+    binutils \
+    binutils-mipsel-linux-gnu
+
+COPY qemu.sh /
+RUN /qemu.sh mipsel
+
+COPY qemu-runner base-runner.sh /
+
```

Comment:
Change the below code to:

```Dockerfile
ENV CROSS_SYSROOT=/x-tools/mipsel-unknown-linux-uclibc/mipsel-unknown-linux-uclibc/sysroot/
ENV CARGO_TARGET_MIPSEL_UNKNOWN_LINUX_UCLIBC_LINKER=mipsel-unknown-linux-uclibc-gcc \
    CARGO_TARGET_MIPSEL_UNKNOWN_LINUX_UCLIBC_RUNNER="/qemu-runner mipsel" \
    CC_mipsel_unknown_linux_uclibc=mipsel-unknown-linux-uclibc-gcc \
    CXX_mipsel_unknown_linux_uclibc=mipsel-unknown-linux-uclibc-g++ \
    BINDGEN_EXTRA_CLANG_ARGS_mipsel_unknown_linux_uclibc="--sysroot=$CROSS_SYSROOT" \
    QEMU_LD_PREFIX="$CROSS_SYSROOT" \
    RUST_TEST_THREADS=1 \
    PKG_CONFIG_PATH="/usr/lib/mipsel-unknown-linux-uclibc/pkgconfig/:${PKG_CONFIG_PATH}"
```

And it works. You can confirm by doing the following tests:

```bash
$ CROSS_TARGET_MIPSEL_UNKNOWN_LINUX_UCLIBC_IMAGE=ghcr.io/cross-rs/mipsel-unknown-linux-uclibc-cross:local TARGET=mipsel-unknown-linux-uclibc BUILD_STD=1 STD=1 CPP=1 DYLIB=1 RUN=1 ci/test.sh
```

It passes the full test suite with that: C++, dynamic library, Rust std, and the Qemu runners. `BUILD_STD=1` is required because it's a tier 3 target and Rust does not provide a pre-built standard library for it.